src/components/Multiselect/AvatarSelectOption.vue

```diff
@@ -130,12 +129,26 @@ export default {
 				obj[`data-${this.toKebabCase(key)}`] = this.$attrs[key]
 				return obj
 			}, {})
+		},
+		search() {
+			return this.$parent.search
+		},
+		escapedDisplayName() {
+			return escapeHtml(this.displayName)
+		},
+		escapedDesc() {
+			return escapeHtml(this.desc)
 		}
+
 	},
 
 	methods: {
-		toKebabCase: (string) => {
+		toKebabCase(string) {
 			return string.replace(/([a-z0-9]|(?=[A-Z]))([A-Z])/g, '$1-$2').toLowerCase()
+		},
+		highlightPhrase(text) {
+			if (!this.search.length) return text
+			return text.replace(this.search, `<strong>${this.search}</strong>`)
```

Comment:
This only works for the first occurrence

Reply:
how so?

Reply:
> If pattern is a string, only the first occurrence will be replaced.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace

Reply:
Ah yes! sure :)

Reply:
![Capture d’écran_2019-08-02_17-17-43](https://user-images.githubusercontent.com/14975046/62380458-8c253700-b549-11e9-84a3-3972ebb935fa.png)
Fixed